md/release-notes.md

```diff
@@ -54,6 +54,12 @@ default log level for `com.bonitasoft` and `org.bonitasoft` change from `WARNING
 From this release, upgrade from Community to Subscription is made [a lot easier](upgrade-from-community-to-a-subscription-edition.md#bonita-platform-upgrade):
 all the tricky configuration part is handled automatically by Bonita Runtime at server startup.
 
+#### JAAS authentication is now Enterprise
+
+From version 2021.2, JAAS Authentication is only available in Enterprise edition.  
+If you are updating from a 2021.1 or earlier version, in Community edition, you will need to [change your configuration](BonitaBPM_platform_setup.md) file `authenticationManager-config.properties`
```

Comment:
@abirembaut Do we also need to remove file `jaas-standard.cfg` under tomcat/server/conf ?

Reply:
@educhastenier It is already only in the subscription edition